src/Rx/Weno/pharmacyupdates.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
this script should be moved to interface subdirectory.
Would put it in:
interface/weno/

Reply:
done